command/ca/ca.go

```diff
@@ -85,27 +85,63 @@ $ step ca renew internal.crt internal.key \
 
 // common flags used in several commands
 var (
-	fingerprintFlag = cli.StringFlag{
-		Name:  "fingerprint",
-		Usage: "The <fingerprint> of the targeted root certificate.",
+	acmeFlag = cli.StringFlag{
+		Name: "acme",
+		Usage: `ACME directory <url> to be used for requesting certificates via the ACME protocol.
+Use this flag to define an ACME server other than the Step CA. If this flag is
+absent and an ACME provisioner has been selected then the '--ca-url' flag must be defined.`,
+	}
+
+	acmeContactFlag = cli.StringSliceFlag{
+		Name: "contact",
+		Usage: `<email-addresses> for contact as part of the ACME protocol. These contacts
+may be used to warn of certificate expration or other certificate lifetime events.
+Use the '--contact' flag multiple times to configure multiple contacts.`,
+	}
```

Comment:
Perhaps `The <email-address> ...` so it displays `--contact=email-address` instead of `--contact=email-addresses`